app/views/_pod-template.html

```diff
@@ -7,194 +7,67 @@
-            {{container.resources.requests.cpu | usageWithUnits: 'cpu'}} requested
-          </span>
-        </div>
-      </div>
+<div>
+  <h4>Containers</h4>
 
-      <div row ng-if="detailed && (container.resources.requests.memory || container.resources.limits.memory)" class="icon-row">
-        <div class="icon-wrap">
-          <i class="fa fa-area-chart" aria-hidden="true"></i>
-        </div>
-        <div flex>
-          <span class="pod-template-key">Memory:</span>
-          <span ng-if="container.resources.requests.memory && container.resources.limits.memory">
-            {{container.resources.requests.memory | usageWithUnits: 'memory'}} to {{container.resources.limits.memory | usageWithUnits: 'memory'}}
-          </span>
-          <span ng-if="!container.resources.requests.memory">
-            {{container.resources.limits.memory | usageWithUnits: 'memory'}} limit
-          </span>
-          <span ng-if="!container.resources.limits.memory">
-            {{container.resources.requests.memory | usageWithUnits: 'memory'}} requested
-          </span>
-        </div>
-      </div>
+  <!-- Prompt to add health checks if missing. -->
+  <div ng-if="detailed && addHealthCheckUrl && !(podTemplate | hasHealthChecks)" class="alert alert-info">
+    <span class="pficon pficon-info" aria-hidden="true"></span>
+    <span ng-if="containers.length === 1">This container has no health checks</span>
+    <span ng-if="containers.length > 1">Not all containers have health checks</span>
+    to ensure your application is running correctly.
+    <a ng-href="{{addHealthCheckUrl}}" class="nowrap">Add Health Checks</a>
+  </div>
 
-      <div row ng-if="detailed && container.readinessProbe" class="icon-row">
-        <div class="icon-wrap">
-          <i class="fa fa-medkit" aria-hidden="true"></i>
-        </div>
-        <div flex>
-          <span class="pod-template-key">Readiness Probe:</span>
-          <probe probe="container.readinessProbe"></probe>
-        </div>
-      </div>
+  <div class="pod-template-container">
+    <div class="pod-template-block" ng-repeat="container in containers | orderBy:'name'">
 
-      <div row ng-if="detailed && container.livenessProbe" class="icon-row">
-        <div class="icon-wrap">
-          <i class="fa fa-medkit" aria-hidden="true"></i>
-        </div>
-        <div flex>
-          <span class="pod-template-key">Liveness Probe:</span>
-          <probe probe="container.livenessProbe"></probe>
-        </div>
-      </div>
+      <pod-template-container
+        pod-template-container="container"
+        images-by-docker-reference="imagesByDockerReference"
+        builds="builds"
+        detailed="detailed"
+        label-prefix="Container"></pod-template-container>
 
       <div
         extension-point
         extension-name="container-links"
         extension-types="link dom"
-        extension-args="[container, podTemplate]"></div>
+        extension-args="[container, containers]"></div>
```

Comment:
This can't change as it might break extensions. We should still pass `podTemplate` as the second arg.